### PR TITLE
Update CaptainManager.ts

### PR DESCRIPTION
**Pull request by uhlhosting**
Updated SMTP/PushBullet/Telegram/Slack and added PushOver experimental. Need to double check values.

First of all, thank you for your contribution! 😄

Please make sure you have read [contribution guidelines](https://github.com/caprover/caprover/blob/master/CONTRIBUTING.md#before-contributing). 

### Most important items

- Make sure to communicate your proposed changes ob our Slack channel beforehand.
- Large PRs (50+ lines of code) will get rejected due to increased difficulty for review - unless they have been communicated beforehand with project maintainers.
- **Refactoring work will get rejected** unless it's been communicated with project's maintainers **beforehand**. There is a thousand ways to write the same code. Every time a code is changed, there is a potential for a new bug. We don't want to refactor the code just for the sake of refactoring.

These rules are strictly enforced to make sure that we can maintain the project moving forward.

### CODE REVIEW

**Comment by uhlhosting** [comment]
@githubsaturn please check it and make adjustments as needed.

**Review by uhlhosting — COMMENTED** [review]
Please double check!

File: src/user/system/CaptainManager.ts [hunk 1]
```diff
@@ -637,8 +647,13 @@ class CaptainManager {
                             key: 'TELEGRAM_CHAT_ID',
                             value: netDataInfo.data.telegram.chatId,
                         })
+                        envVars.push({
+                            key: 'SEND_TELEGRAM',
+                            value: 'YES',
```
Comment (githubsaturn) [inline src/user/system/CaptainManager.ts:652]: These are redundant. They can be set to YES in the `run.sh` file of the netdata container if the parent env var (e.g.,  TELEGRAM_CHAT_ID) is present.

File: src/user/system/CaptainManager.ts [hunk 2]
```diff
@@ -648,8 +663,44 @@ class CaptainManager {
...
+                            key: 'SEND_PUSHBULLET',
+                            value: 'YES',
+                        })
                     }
-
+                    
+                    // For Discord Block
+                    if (netDataInfo.data.discord) {
+                        envVars.push({
+                            key: 'DISCORD_WEBHOOK_URL',
```
Comment (githubsaturn) [inline src/user/system/CaptainManager.ts:675]: The new env vars need to be reflected in the frontend as well. They won't be used if they're not surfaced on the frontend.

**Review by githubsaturn — COMMENTED** [review]
Thanks for the PR! Just a few comments. Also, it looks like the build is failing. Please fix the issues and resubmit.

File: src/user/system/CaptainManager.ts (hunk 1 above)
Comment (uhlhosting) [reply src/user/system/CaptainManager.ts:652]: Ok

File: src/user/system/CaptainManager.ts (hunk 2 above)
Comment (uhlhosting) [reply src/user/system/CaptainManager.ts:675]: They were present in the run.sh I was thinking they must be also in the frontend.

File: src/user/system/CaptainManager.ts (hunk 1 above)
Comment (uhlhosting) [reply src/user/system/CaptainManager.ts]: So with last changes, should be good now, give it a go.

**Comment by githubsaturn** [comment]
Thanks! But this is a no-op change now. I'll close, feel free to re-add if things change.

image

**Comment by uhlhosting** [comment]
Well the issue is I am not sure how you use these values. As to what I see in run.sh there are all new variables there. And since you said yes can be defined in run.sh. It was defined there only. So it seems for now no changes might be required here.

**Pull request closed without merging**